commands/channel.go

```diff
@@ -356,18 +357,23 @@ func unarchiveChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) e
 		return errors.New("enter at least one channel")
 	}
 
+	var multErr *multierror.Error
 	channels := getChannelsFromChannelArgs(c, args)
 	for i, channel := range channels {
 		if channel == nil {
-			printer.PrintError("Unable to find channel '" + args[i] + "'")
+			err := fmt.Errorf("Unable to find channel '%s'", args[i])
+			printer.PrintError(err.Error())
+			multErr = multierror.Append(multErr, err)
 			continue
 		}
 		if _, _, err := c.RestoreChannel(channel.Id); err != nil {
-			printer.PrintError("Unable to unarchive channel '" + args[i] + "'. Error: " + err.Error())
+			restoreError := fmt.Sprintf("Unable to unarchive channel '" + args[i] + "'. Error: " + err.Error())
```

Comment:
You can directly create this as a `fmt.Errorf`, this will also trigger `Error: ST1005: error strings should not be capitalized (stylecheck)`.